sparse/_coo/numba_extension.py

```diff
@@ -52,7 +52,7 @@ def coords_type(self):
 
     @property
     def shape_type(self):
-        return types.UniTuple(types.int64, self.ndim)
+        return types.UniTuple(types.intp, self.ndim)
```

Comment:
I'd prefer not to hard-code.
```suggestion
        dt = numba.np.numpy_support.from_dtype(self.coords_dtype)
        return types.UniTuple(at, self.ndim)
```